050-Manage/030-migrations.mdx

```diff
@@ -73,10 +73,131 @@ Migration files should not be modified. If externally modified, the record will
 
 Migrations must be ordered. Ordering is guaranteed by the `parent` and the ledger file, which is stored in `.xata/migrations/.ledger`. The ledger file is an append only file that lists all migration files in the correct order.
 
-## UI Migration Editor for pgroll operations
+## Multi-version Schema Migrations (Postgres enabled only)
 
-Under the Schema menu, the Web UI provides a Migration Editor to create migrations using [pgroll](https://github.com/xataio/pgroll) operations.
+For Postgres enabled branches, the Web UI provides a Migration Editor to create [multi-version schema migrations](https://xata.io/blog/schema-multi-version-launch-week) using [pgroll](https://github.com/xataio/pgroll) operations.
```

Comment:
```suggestion
For Postgres enabled branches, Xata provides a Migration Editor to create [multi-version schema migrations](https://xata.io/blog/schema-multi-version-launch-week) using [pgroll](https://github.com/xataio/pgroll) operations.
```

Trying to avoid using "UI" in the docs. There is a decision issue somewhere, but I can't find it off hand. I think we landed on calling it the "Xata app".

Reply:
It may also be worth providing a little more context early on when you'd want to use this type of migration. Something like:

"For simple schema migrations or if downtime is acceptable, you can edit your schema directly in the Table Editor or Schema Editor. For more complex ones, you should consider multi-version schema migrations. 

For Postgres enabled branches..."

Just an idea, but that felt like it might be explicitly worth clarifying